app/assets/javascripts/form_exit.js.erb

```diff
@@ -1,9 +1,14 @@
 ;(function (w, d) {
   exitHandler = function () {
     var links = d.querySelectorAll('A');
+    var inSteps = window.location.href.indexOf('/steps/') >= 0;
+    if(!inSteps) return;
 
     for(var i=0; i<links.length; i++){
-      if(links[i].href.indexOf('steps') < 0){
+      var notStepsLink = links[i].href.indexOf('steps') < 0
+      var notSkipLink = links[i].href.indexOf('skip_') < 0
+
+      if(notStepsLink && notSkipLink){
```

Comment:
Take into account the situation where, on the very last page, we're clicking the "skip" link. We don't want to be alerted if we click the "skip" link,